apps/www/pages/blog/[slug].tsx

```diff
@@ -124,10 +124,13 @@ function BlogPostPage(props: any) {
     </div>
   )
 
+  console.log('props.blog', props.blog)
+
   return (
     <>
       <NextSeo
         title={props.blog.title}
+        description={props.blog.description}
```

Comment:
Just want to confirm this is the correct description from the blog file?

Reply:
yep